special-widget.scss

```diff
@@ -0,0 +1,17 @@
+.special-widget {
+	background: grey;
```

Comment:
It might be worth setting up a theming solution that defines a set of colour variables used across the project, these can be imported into SCSS files. The approach would provide consistency and ease of theming at a later date making the project more maintainable. I can point you to some examples if you need any further information.